From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 17 Sep 2018 16:08:46 +0200
Subject: Revert "HACK: etnaviv: hack around "BO stride is too small for RS
 engine width padding""

Not needed anymore due to the previous commit.

This reverts commit d00616eaf563edd33be1aa6c1db5481e79c3bd49.
---
 src/gallium/drivers/etnaviv/etnaviv_resource.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gallium/drivers/etnaviv/etnaviv_resource.c b/src/gallium/drivers/etnaviv/etnaviv_resource.c
index acc678b..8facdc1 100644
--- a/src/gallium/drivers/etnaviv/etnaviv_resource.c
+++ b/src/gallium/drivers/etnaviv/etnaviv_resource.c
@@ -545,13 +545,13 @@ etna_resource_from_handle(struct pipe_screen *pscreen,
       BUG("BO stride %u is too small for RS engine width padding (%zu, format %s)",
           level->stride, util_format_get_stride(tmpl->format, level->padded_width),
           util_format_name(tmpl->format));
-      // goto fail;
+      goto fail;
    }
    if (etna_bo_size(rsc->bo) < level->stride * level->padded_height) {
       BUG("BO size %u is too small for RS engine height padding (%u, format %s)",
           etna_bo_size(rsc->bo), level->stride * level->padded_height,
           util_format_name(tmpl->format));
-      // goto fail;
+      goto fail;
    }
 
    if (rsc->layout == ETNA_LAYOUT_LINEAR) {
